### DISCUSSION TITLE
Manage Card Fields is not showing any data

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2799] [Q&A]
I have the following setting inside my PnP - Search Result : -

![image](https://user-images.githubusercontent.com/79726511/220984929-e7189c60-0bf2-407b-8c16-7c3311fd41d0.png)

![image](https://user-images.githubusercontent.com/79726511/220985285-e57f74c0-e13a-4d40-a7e5-b958bf428dc0.png)

![image](https://user-images.githubusercontent.com/79726511/220985030-a58349c9-03e5-495d-989d-1c6c47b5c4d8.png)

![image](https://user-images.githubusercontent.com/79726511/220985217-48542a51-bfdf-485d-8185-ce347724b9f4.png)

so i chose the Cards layout >> and i define to show FileName + Path >> but the cards will be shown as empty (except for the document icon).. any advice please?

![image](https://user-images.githubusercontent.com/79726511/220985762-48a6e466-81bf-4c6c-81f1-d6052eb31697.png)

**Comment by gretchunkim** [comment]
@mvcsharepointdev 

You're getting 9 results, which means it's the labeling item property issue. 

To troubleshoot, it's best to duplicate that webpart then pick debug mode. then you'll see which properties you can use to populate them in your original webpart. On your original webpart, uncheck handlebar and directly pick properties that you see from debug.
